src/test/regress/expected/ref_citus_local_fkeys.out

```diff
@@ -196,10 +184,7 @@ NOTICE:  executing the command locally: SELECT worker_apply_inter_shard_ddl_comm
 -- tables works fine with remote execution too
 SET citus.enable_local_execution TO OFF;
 ALTER TABLE reference_table DROP CONSTRAINT fkey_ref_to_local;
-NOTICE:  creating a new table for ref_citus_local_fkeys.citus_local_table
 NOTICE:  Moving the data of ref_citus_local_fkeys.citus_local_table
```

Comment:
It seems Moving notice is skipped?

Reply:
I added the same notice message as it might take same time
https://github.com/citusdata/citus/pull/4567/files#diff-54932819bfb664fe87c889e8b8241493be36fdc3e0cb791ecc9c20adbb861828R732-R742

Reply:
e908750